composer.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "reload/bugyield",
+  "description": "A script to sync between Harvest and Jira.",
+  "license": "MIT",
+  "autoload" : {
+    "psr-4": {
+      "BugYield\\": "src/BugYield"
```

Comment:
This is in effect the same as before - just overly verbose.

My suggestion was to change it to "BugYield\\": "src/" and then move all source up one level and remove the BugYield directory.